pkg/frontend/frontend.go

```diff
@@ -475,6 +475,14 @@ func frontendOperationResultLog(log *logrus.Entry, method string, err error) {
 
 	switch err := err.(type) {
 	case *api.CloudError:
+		if int(err.StatusCode) < 300 && int(err.StatusCode) >= 200 {
```

Comment:
Can we really have a "successful" CloudError?

Reply:
Considering the condition, there could not be a "real" error but a CloudError that has a statusCode between 200 & 300. 
Eg : https://github.com/Azure/ARO-RP/blob/master/pkg/frontend/admin_hive_clusterdeployment_get.go#L45

Reply:
An error with a 204 seems like a bug to me and IMO we should make that impossible. Definitely not for this PR though. 

Also after a quick grep I think you might have the only instance of the a CloudError with a non 4XX/5XX status code in the whole RP so maybe the change I'm implying wouldn't be so painful. If you have sometime maybe we could pair on this as a subsequent task? I don't think I've met any of the APAC folks yet.